charts/s3gw/templates/configmap.yaml

```diff
@@ -7,16 +7,6 @@ metadata:
   labels:
 {{ include "s3gw.labels" . | indent 4}}
 data:
-{{- if .Values.ingress.enabled }}
-  RGW_DNS_NAME: >-
-    {{ .Values.serviceName }}.{{ .Values.publicDomain }},
-    {{ .Values.serviceName }}.{{ .Release.Namespace }}.{{ .Values.privateDomain }}
-{{- else}}
-  RGW_DNS_NAME: >-
-    {{ .Values.serviceName }}.{{ .Release.Namespace }}.{{ .Values.privateDomain }}
-{{- end }}
-  RGW_BACKEND_STORE: "sfs"
-  DEBUG_RGW: '{{ .Values.logLevel }}'
```

Comment:
Changes RGW_BACKEND_STORE and DEBUG_RGW not mentioned in the commit message. Why?

Reply:
Consider that `configmap.yaml` was initially introduced when we were still not using charts.
Now, the real usefulness of this file is relative; we can directly use the values from the charts and simplify the overall structure.
I think we can consider this as an implementation detail; btw if you think it is worth to mention these changes on the commit message I will add it.

Reply:
done